sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -333,8 +341,8 @@ public:
   command_graph<graph_state::executable>
   finalize(const context& syclContext, const property_list& propList = {}) const;
 
-  bool begin_recording(queue& recordingQueue);
-  bool begin_recording(const std::vector<queue>& recordingQueues);
+  bool begin_recording(queue& recordingQueue, const property_list& propList = {});
+  bool begin_recording(const std::vector<queue>& recordingQueues, const property_list& propList = {});
```

Comment:
Realised the signatures in the table of queue recording member functions will need updated too

Reply:
Good point, I've added them now!